filters/openpolicyagent/openpolicyagent.go

```diff
@@ -557,7 +571,7 @@ func (registry *OpenPolicyAgentRegistry) new(store storage.Store, configBytes []
 
 	opa := &OpenPolicyAgentInstance{
 		registry:       registry,
-		instanceConfig: instanceConfig,
+		instanceConfig: *registry.configTemplate,
```

Comment:
This assumes registry.configTemplate is not nil which implies/requires that WithOpenPolicyAgentInstanceConfig has been called.
Can we check that it is not nil and return an error before instead of potentially failing with nil dereference?

Reply:
Good catch. Let me address this...